requirements.txt

```diff
@@ -0,0 +1,17 @@
+ddt
+lxml
+mock
+selenium
+pep8
+pylint
+coverage==3.7
+coveralls
+django==1.4.22
```

Comment:
Just be aware that we are in the midst of a massive Django 1.8 upgrade so we'll have to swing back to this. FYI @nedbat @macdiesel